database/yugabytedb/yugabytedb.go

```diff
@@ -0,0 +1,427 @@
+package yugabytedb
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/url"
+	"regexp"
+	"strconv"
+
+	"github.com/golang-migrate/migrate/v4"
+	"github.com/golang-migrate/migrate/v4/database"
+	"github.com/hashicorp/go-multierror"
+	"github.com/lib/pq"
```

Comment:
Any reason why you chose to use the pq driver instead of [their version](https://docs.yugabyte.com/preview/reference/drivers/go/yb-pgx-reference/) of the pgx driver?

Reply:
The only difference between these drivers is load-balancing support. I think we don't need load-balancing for running migrations. Even more, load-balancing can cause problems in the migrations' scenario...